keel-front50/src/main/kotlin/com/netflix/spinnaker/keel/front50/model/Application.kt

```diff
@@ -17,18 +17,20 @@ package com.netflix.spinnaker.keel.front50.model
 
 import com.fasterxml.jackson.annotation.JsonAnyGetter
 import com.fasterxml.jackson.annotation.JsonAnySetter
+import com.netflix.spinnaker.keel.front50.annotations.Computed
 
 data class Application(
   val name: String,
   val description: String,
   val email: String,
-  val updateTs: String,
-  val createTs: String,
+  @Computed val updateTs: String = "",
+  @Computed val createTs: String = "",
```

Comment:
Do you think we should just make these nullable instead?

Reply:
When we give the params a default value we don't have to pass them into the constructor as 'null'. Seems nicer, for testing and creating new things? I don't particularly care though.

Reply:
I'd rather see them be `String?` signatures with default values of `null`. At least with null, we know the value isn't valid because it isn't set.